modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm

```diff
@@ -85,6 +85,18 @@
 	default_color = "#CCCCCC"
 	affected_bodyparts = HEAD
 
+/datum/body_marking/other/monster2_mouth
+	name = "Monster Mouth (White)"
+	icon_state = "monster2"
+	default_color = "#CCCCCC"
+	affected_bodyparts = HEAD
+
+/datum/body_marking/other/monster3_mouth
+	name = "Monster Mouth (White, eyes-compatible)"
+	icon_state = "monster3"
+	default_color = "#CCCCCC"
+	affected_bodyparts = HEAD
+
```

Comment:
Move these to a folder under modular_zubbers, and link the icon file with `icon='filepath'`